build.gradle

```diff
@@ -44,9 +44,6 @@ allprojects {
 }
```

Comment:
Can we also move `run` tasks to dedicated modules?

Reply:
It can be done, but it will lead to either moving all related configs to modules or addition paths processing (in every file/method related to configs, keys etc) since we need to work with 'parent' directory from a module.
So the easiest (and probably right) way is to have gradle tasks on the same level with the related materials. (Both configs and tasks at the root OR both configs and tasks in the corresponding module).
Should we do such refactoring and if yes what is the exact strategy? @Alexey-N-Chernyshov @dolgopolovwork

Reply:
Please take a look at `btc` services. We can do it with ease.

Reply:
Thank you for the advice
I hope I did it correctly